tests/npm-app/src/endpoints/snp_attestation.ts

```diff
@@ -0,0 +1,123 @@
+import { Base64 } from "js-base64";
+
+import * as ccfapp from "@microsoft/ccf-app";
+import * as ccfsnp from "@microsoft/ccf-app/snp_attestation";
+
+interface ErrorResponse {
+  error: {
+    message: string;
+  };
+}
+
+interface SnpEvidence {
+  evidence: string;
+  endorsements: string;
+  endorsed_tcb?: string;
+}
+
+export interface TcbVersion {
+  boot_loader: number;
+  tee: number;
+  snp: number;
+  microcode: number;
+}
+
+interface SnpAttestationResult {
```

Comment:
why is this duplicated and not imported from ccf-app?

Reply:
In ccf-app, the values are array buffers, but here we convert them into hex strings so those types have changed, its the same pattern we use for the OE bindings